server/modules/auth/repository/jwtRepository.js

```diff
@@ -0,0 +1,28 @@
+const db = require('../../../db/db')
+
+const blacklistToken = async (token, expiration, client = db) => {
+  client.none(`
+    INSERT INTO jwt_token_blacklist (token, expiration)
+    VALUES ($1, $2)
+  `, [token, expiration])
+}
+
+const findBlacklistedToken = async (token, client = db) =>
+  client.oneOrNone(`
```

Comment:
do not leave pending promises. alway add ```await```

Reply:
Ok